purview/models/errata.py

```diff
@@ -47,3 +47,14 @@ class AdvisoryState(PurviewStructuredNode):
     name = StringProperty(required=True)
     advisory = RelationshipTo('Advisory', 'STATE_OF', cardinality=ZeroOrOne)
     creator = RelationshipTo('.user.User', 'CREATED_BY', cardinality=ZeroOrOne)
+
+
+class Product(PurviewStructuredNode):
```

Comment:
After thinking about it, this should likely be called `ProductVersion`.

Reply:
hm, I am more inclined towards Product, since it's not explicitly for storing only the version_info.